plugin/src/main/java/com/epam/grandhackathon/deployment/sphere/plugin/BuildVersionMetaDataCollectorDescriptor.java

```diff
@@ -54,4 +61,13 @@ public FormValidation doCheckAppName(@QueryParameter String appName) throws IOEx
         return FormValidation.ok();
     }
 
+	public ListBoxModel doFillAppNameItems() {
+		Collection<ApplicationMetaData> applications = applicationDao.findAll();
+		ListBoxModel items = new ListBoxModel();
+		for (ApplicationMetaData option : applications) {			
```

Comment:
With Guava without changing ListBoxModel I cant convert from com.google.common.collect.Collections2$TransformedCollection to hudson.util.ListBoxModel. Please see class ListBoxModel